protocol/flows/ibd/ibd.go

```diff
@@ -227,6 +235,10 @@ func processIBDBlock(dag *blockdag.BlockDAG, msgIBDBlock *wire.MsgIBDBlock) (sho
 		return false, protocolerrors.Errorf(false, "received delayed block %s "+
 			"during IBD", block.Hash())
 	}
+	err = newBlockHandler(block)
```

Comment:
I'd rather if newBlockHandler was called blockAddedHandler or something like that, newBlockHandler sounds like a constructor